src/command_modules/azure-cli-dla/azure/cli/command_modules/dla/custom.py

```diff
@@ -4,6 +4,11 @@
 # --------------------------------------------------------------------------------------------
 import time
 import uuid
+# to ensure we have consistent access to the quote() method
+try:
+    from urllib import parse as safe_url_lib
+except ImportError:
+    import urllib as safe_url_lib
```

Comment:
@derekbekoe interesting! How do I use it for urllib? I am confused by the documentation there what the actual module is called in six, and VS intellisense is complaining that there is no urlib.parse.quote() functionality anywhere in this module.